hidtest/test.c

```diff
@@ -80,7 +81,7 @@ int main(int argc, char* argv[])
 				if (i % 10 == 0) {
 					printf("\n");
 				}
-				printf("0x%02x, ", buf[i]);
+				printf("0x%02X, ", buf[i]);
```

Comment:
any particular reason to use upper-case?

Reply:
Only readability of the test output. For me it was easier to read a list like 0xA6, 0xC0 than 0xa6, 0xc0.
If you prefer the other style, feel free to change it.

Reply:
Once written and working, I prefer not to change anything unless necessary.
Smaller diff is always preferable, when possible.

Reply:
```suggestion
				printf("0x%02x, ", buf[i]);
```